explore-reports-and-create-documentation/farm-explorer/farm-dashboard.md

```diff
@@ -1,2 +1,51 @@
+---
+title: Farm Dashboard
+description: This article describes Farm Dashboard and its features.
+Author: Janko Ban
+date: 14/12/2018.
+---
+
 # Farm Dashboard
 
+Farm dashboard is giving you an overview of the farm configuration, drillable to specific detailed reports.
+
+The following farm configurations are visible:
+
+* Number of servers, SQL Servers, Web Applications as well as Site Collections
+* Top Databases by Total Size
+* Top Site Collections by Total Size
+* Farm Accounts
+* Service Applications
+* Health Analyzer Problems
+* Farm Solutions Deployed
+* Best Practices
+
+Number of __Servers, SQL Servers__, __Web Applications__, and __Site Collections__ are represented separately in its tile, respectively. Clicking on the tile performs a drill to a detailed report.
+
+Drill opens the following reports:
+
+* Servers          --> Servers List
```

Comment:
tu umjesto ovih strelica napiši Servers tile drill to the Servers List report itd.

Reply:
isto i dolje gdje je takvo pobrojavanje